### ISSUE TITLE
about nii2png

### ISSUE THREAD

**Issue by iaOlivia** [lpcccc-cv/MC-CDic#5]
Hi,
for IXI-T2 datasets ,it contains 578 samples,set volume_num = 500 mean skips first 500 samples only use remain 77(except ixi4）samples? if it's true,why skip most samples(500) only keeps the last 78 samples?It seems for me a little unreasonable.
Or volume_num for ixi-T2 datasets should set another value?
Please help with this.

**Comment by iaOlivia** [comment]
below is code segment makes me confused.
**volume_num = 500  #** set the first 500 volumes as the training data
totle_number=0  # total image number

file_list = sorted(os.listdir(file_path))

for number, name in enumerate(file_list):
    if 'IXI014' in name:
        continue  # skip it
    # change
    **if number<volume_num:
        continue**
    print(name)
    filename_1 = file_path+'/'+name
    img_1 = sitk.ReadImage(filename_1, sitk.sitkInt16)
    space = img_1.GetSpacing()
    img_1 = sitk.GetArrayFromImage(img_1)
    width, height, queue = img_1.shape

**Comment by lpcccc-cv** [comment]
No, it just an example. We use all the volumes for our experiments. You can set the volume_num to choose what data you want for training, validation or  testing.



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/10/2023 12:06 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [lpcccc-cv/MC-CDic] about nii2png (Issue #5) |

Hi,
for IXI-T2 datasets ,it contains 578 samples,set volume_num = 500 mean skips first 500 samples only use remain 77(except ixi4）samples? if it's true,why skip most samples(500) only keeps the last 78 samples?It seems for me a little unreasonable.
Or volume_num for ixi-T2 datasets should set another value?
Please help with this.

**Comment by iaOlivia** [comment]
OK.I see.
